app/views/password_resets/edit.html.erb

```diff
@@ -0,0 +1,29 @@
+<!-- https://gist.github.com/wendygwo/6d65db594151c7a9d459 -->
+<div class="container whiteboard">
+  <div class="row">
+    <h1>Reset Password</h1>
+  </div>
+  <hr>
+  <div class="row">
+    <div class="from-group">
+      <%= form_for @user, :url => password_reset_path(params[:id]) do |f| %>
+        <%= render "layouts/_error_messages" %>
+
+        <div class='field'>
+          <%= f.label :password %>
+          <%= f.password_field :password, class: "form-control", required: true  %>
+        </div>
+        <div class='field'>
+          <%= f.label :password_confirmation %>
+          <%= f.password_field :password_confirmation, class: "form-control", required: true  %>
+        </div>
+        <div class='actions'><%= f.submit 'Update password', class: "btn btn-primary" %></div>
+      <% end %>
+    </div>
+    <div>
+      <% if flash[:notice] %>
+        <div class="notice"><%= flash[:notice] %></div>
+      <% end %>
```

Comment:
Diesen Code kannst du doch auslagern? Das ist dasselbe wie in `new.html.erb`.

Reply:
geändert